src/common/transformations/src/transformations/convert_precision.cpp

```diff
@@ -554,6 +557,18 @@ bool fuse_type_to_unique_v10(const std::shared_ptr<Node>& node, const precisions
     return res;
 }
 
+bool fuse_type_to_serach_sorted_v15(const std::shared_ptr<Node>& node, const precisions_map& precisions) {
```

Comment:
please add a test
@CuriousPanCake has already sent a patch with a new test, you can just apply it

Reply:
Fixed.